examples/src/components/GeoPushpinChartConfigurationPointsExample.jsx

```diff
@@ -0,0 +1,149 @@
+
+import { projectId } from "../utils/fixtures";
+import { MAPBOX_TOKEN, cityCoordinatesUri, populationUri } from "../utils/fixturesGeoChart";
+
+const sizeMeasure = Model.measure(populationUri)
+    .format("#,##0.00")
+    .aggregation("sum")
+    .alias("Population");
+const locationAttribute = Model.attribute(cityCoordinatesUri).localIdentifier("location");
+const POINT_SIZE_OPTION = [
+    { value: "default", label: "default" },
+    { value: "0.5x", label: "0.5x" },
+    { value: "0.75x", label: "0.75x" },
+    { value: "normal", label: "normal" },
+    { value: "1.25x", label: "1.25x" },
+    { value: "1.5x", label: "1.5x" },
+];
+
+export class GeoPushpinChartConfigurationPointsExample extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            points: {},
+            togglePinSizes: false,
+        };
+    }
+    onLoadingChanged(...params) {
+        // eslint-disable-next-line no-console
+        return console.log("GeoPushpinChartConfigurationPointsExample onLoadingChanged", ...params);
+    }
+
+    onError(...params) {
+        // eslint-disable-next-line no-console
+        return console.log("GeoPushpinChartConfigurationPointsExample onError", ...params);
+    }
+
+    onZoomChanged(...params) {
+        // eslint-disable-next-line no-console
+        return console.log("GeoPushpinChartConfigurationPointsExample onZoomChanged", ...params);
+    }
+
+    onCenterPositionChanged(...params) {
+        // eslint-disable-next-line no-console
+        return console.log("GeoPushpinChartConfigurationPointsExample onCenterPositionChanged", ...params);
+    }
+
+    onPointSizeChange = event => {
+        const { id, value } = event.target;
+        this.setState(prevState => {
+            return {
+                ...prevState,
+                points: {
+                    ...prevState.points,
+                    [id]: value,
+                },
+            };
+        });
+    };
+
+    togglePinSizes = () => {
+        this.setState(prevState => {
+            return {
+                ...prevState,
+                togglePinSizes: !prevState.togglePinSizes,
+            };
+        });
+    };
+
+    toggleGroupNearbyPoints = () => {
+        this.setState(prevState => {
+            return {
+                ...prevState,
+                points: {
+                    ...prevState.points,
+                    groupNearbyPoints: !prevState.points.groupNearbyPoints,
+                },
+                togglePinSizes: false,
+            };
+        });
+    };
+
+    renderPointSizeDropDown = id => {
+        return (
+            <span style={{ display: "inline-block", minWidth: "10em", verticalAlign: "middle" }}>
+                {id}
+                <select id={id} onChange={this.onPointSizeChange}>
+                    {POINT_SIZE_OPTION.map((size, index) => (
+                        <option key={id + index.toString()} value={size.value}>
```

Comment:
key could be value or label is enough IMO

Reply:
fixed